validateJR.sh

```diff
@@ -16,21 +16,27 @@ grep root ${inList} | grep -v "#" | while read -r dsN fN procN comm; do
     cp ~/tools/validate.C ./
     echo "Will run on ${fN} in ${cWD}/${extN}"
     echo "Now in `pwd`"
-#    g++ -shared -o validate.so validate.C `root-config --cflags ` -fPIC
-    echo -e "gSystem->Load(\"libFWCoreFWLite.so\");\n FWLiteEnabler::enable();\n 
-    .x validate.C+(\"${extN}\", \"${baseA}/${fN}\", \"${baseB}/${fN}\", \"${procN}\");\n .qqqqqq" | root -l -b >& ${extN}.log &
-# manually set the make flags, not needed in most cases
-#    gSystem->SetMakeSharedLib(\"cd \$BuildDir ;g++  -c \$Opt -pipe -m64 -Wshadow -Wall -W -Woverloaded-virtual -fPIC -std=c++11 -Wno-deprecated-declarations -DG__MAXSTRUCT=36000 -DG__MAXTYPEDEF=36000 -DG__LONGLINE=4096 -pthread \$IncludePath \$SourceFiles ; g++ \$ObjectFiles -shared -Wl,-soname,\$LibName.so -m64 -Wl,--hash-style=gnu -O2  \$LinkedLibs -o \$SharedLib\");\n 
-#    cout<< gSystem->GetMakeSharedLib()<<endl ;\n
+    #g++ -shared -o validate.so validate.C `root-config --cflags ` -fPIC
+    if [ -f "${baseA}/${mFN}" ]; then 
+        echo -e "gSystem->Load(\"libFWCoreFWLite.so\");\n FWLiteEnabler::enable();\n 
+        .x validate.C+(\"${extN}\", \"${baseA}/${fN}\", \"${baseB}/${fN}\", \"${procN}\");\n 
+        .x validate.C+(\"${extN}\", \"${baseA}/${mFN}\", \"${baseB}/${mFN}\", \"${mProcN}\");\n .qqqqqq" | root -l -b >& ${extN}.log &
+    else
+        echo -e "gSystem->Load(\"libFWCoreFWLite.so\");\n FWLiteEnabler::enable();\n 
+        .x validate.C+(\"${extN}\", \"${baseA}/${fN}\", \"${baseB}/${fN}\", \"${procN}\");\n .qqqqqq" | root -l -b >& ${extN}.log &
+    fi
```

Comment:
By default, I expect both to be processed as long as the files are present.
We should probably put them in the same directory but with an extra extension in the name.
So, use extmN = all_mini_${diffN}_${dsN} for the miniAOD.

There should be a flag to disable miniAOD, probably just $5 "noMINIAOD" or something.